Rakefile

```diff
@@ -12,7 +12,7 @@ end
 
 desc 'Start server'
 task :server do
-  sh "jekyll serve --watch --incremental"
+  sh "jekyll serve --no-watch --incremental"
```

Comment:
What's the reason for this change?

Reply:
me commiting too much :)

Reply:
(watch would bomb out on my machine for some reason, blame inotify, so I disabled it)